### PR TITLE
Adds Project page variable in action output

### PR DESCRIPTION
**Pull request by Maker-Mark**
### Linked issue
Provide the issue(s) which this pull request relates to or fixes.
#11 

### Additional context
At the end of builds, the output includes a Project page where the QR code is available and editable, depending on what client you want it to run on (dev or expo go).
 ### Testing
You can test the action from my fork at https://github.com/marketplace/actions/expo-github-action-fork
Tested on my own private repos, and is particularly helpful when the QR codes don't work and you need a link.

### CODE REVIEW

**Review by lukmccall — CHANGES_REQUESTED** [review]
Thanks for your contribution 🚀 Looks great but could you replace tabs with 2 spaces in your code?

**Comment by Maker-Mark** [comment]
@lukmccall Sure thing. I just pushed my changes with 2 spaces instead of tabs, removed the added double quote, and the import reordering. The diff should be just the intended changes now. 
Let me know if there's anything else you need to get this merged and released!

File: src/publish.ts [hunk 1]
```diff
@@ -12,15 +11,28 @@ export function findManifestUrl(text: string): string {
...
 	);
 	if (exitCode !== 0) {
 		throw new Error(stderr);
 	}
+	const manifestURL = findManifestUrl(stdout);
+	const projectPageURL = findProjectPage(stdout);
 
-	return findManifestUrl(stdout);
+	return { manifestURL, projectPageURL };
 }
```
Comment (lukmccall) [inline src/publish.ts:38]: This file still uses a tabulation 😅

File: tests/publish.test.ts [hunk 2]
```diff
@@ -1,5 +1,49 @@
...
+		expect(projectPageUrl).toBe('https://expo.dev/@markwas/here?release-channel=pr-3');
+	});
+
+	it('should throw if cannot find project page URL', () => {
+		expect(() => {
+			const expoPublishOutput = 'nothing to see here...';
+			findManifestUrl(expoPublishOutput);
+		}).toThrow();
+	});
+});
```
Comment (lukmccall) [inline tests/publish.test.ts:46]: Same here

**Review by lukmccall — CHANGES_REQUESTED** [review]

File: tests/publish.test.ts (hunk 2 above)
Comment (Maker-Mark) [reply tests/publish.test.ts]: @lukmccall I figured it out! The`.pretteirrc` was missing ` "useTabs": false,` so when saving it was using tabs no matter what.

File: src/publish.ts (hunk 1 above)
Comment (Maker-Mark) [reply src/publish.ts]: Fixed it!

File: tests/publish.test.ts (hunk 2 above)
Comment (lukmccall) [reply tests/publish.test.ts]: I have an extension that marks spaces as `.` and tabs as `->` 😅

**Comment by Maker-Mark** [comment]
@lukmccall I fixed the tabs issue now, I promise!

**Review by lukmccall — APPROVED** [review]
Let's wait for a CI if everything is okay, I'll merge it ;)

**Comment by lukmccall** [comment]
Thanks for your contribution 🚀

**Comment by Maker-Mark** [comment]
@lukmccall Awesome! Now I can use the official version for a PR I have open on my own project. Appreciate the quick reviews. 
I'm happy to continue to contribute to this and be a reviewer, if there's anything that needs work on, feel free to reach out to me, I stay on top of my github mentions.

**Comment by thespacemanatee** [comment]
@Maker-Mark @lukmccall I'm getting `Error: Couldn't extract project page URL.` which arises from this commit, any idea why?

**Pull request merged**